fab/services/templates/celery_bash_runner.sh

```diff
@@ -14,7 +14,7 @@
 # Effectively we've orphaned
 # the Celery process to do a warm shutdown and we are
 # free to start another bash process under supervisor.
-trap 'kill -TERM $PID' TERM INT
+trap 'echo "Killing: $PID"; kill -TERM $PID; echo "Killed: $PID";' TERM INT QUIT
```

Comment:
note i added QUIT here in case for some reason supervisor got anxious and called SIGQUIT